app/Platform/Controllers/DeveloperFeedController.php

```diff
@@ -73,15 +73,15 @@ private function attachRecentLeaderboardEntryRowsMetadata(mixed $entryRows): mix
         $entryRows->transform(function ($row) use ($userData, $gameData, $consoleData) {
             $game = $gameData[$row->GameID] ?? null;
 
-            $row->User = $userData[$row->UserID]->User ?? null;
-            $row->Untracked = $userData[$row->UserID]->Untracked ?? null;
+            $row->User = $userData[$row->user_id]->User ?? null;
```

Comment:
Users aren't appearing in recent leaderboard entries on the developer feed.

Reply:
Thanks, I saw the rows populating but I didn't look closely enough to realize users were missing. The `pluck()` call needed to be changed to target the right field. Resolved in latest.